doc/howto/network_forwards.md

```diff
@@ -49,19 +50,23 @@ Property         | Type       | Required | Description
 `ports`          | port list  | no       | List of {ref}`port specifications <network-forwards-port-specifications>`
 
 (network-forwards-listen-addresses)=
+
 ### Requirements for listen addresses
 
 The requirements for valid listen addresses vary depending on which network type the forward is associated to.
 
 Bridge network
 : - Any non-conflicting listen address is allowed.
-  - The listen address must not overlap with a subnet that is in use with another network.
+
+- The listen address must not overlap with a subnet that is in use with another network.
```

Comment:
This is wrong - it messes up the structure. The bullet list is part of the definition list and thus must be indented, without a blank line.